### DISCUSSION TITLE
GitHub Actions

### DISCUSSION THREAD

**Discussion by Digvijayautomation** [mfaisalkhatri/selenium4poc#54] [General]
What changes do we need to do in the maven.yml for running the test after push

**Comment by mfaisalkhatri** [comment]
> What changes do we need to do in the maven.yml for running the test after push

Once you commit and push the code to your branch/main, then it will automatically start the run.

**Reply by Digvijayautomation** [reply]
Yes, but it is not doing the actual executions for test cases.

**Reply by mfaisalkhatri** [reply]
Can you share the link to your repo where you have the workflow file.. I can take a quick look.
